low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -163,8 +168,52 @@ def get_cpu_info(self):
         ]
         return cpu_data
 
+    def get_json_formatted_reading(self, reading):
+        """builds json resposne from ipmi tool response.
+        reading arg sample: ('CPU1 Temp', '01', 'ok', '3.1', '36 degrees C')
+        """
+        uid = '_'.join(reading[0].split())
+        sensor_id = reading[0]
+        sensor_props = self._ipmi.get_sensor_props(sensor_id)
+        lower_critical = sensor_props[1].get('Lower Critical', 'NA')
+        upper_critical = sensor_props[1].get('Upper Critical', 'NA')
+        status = 'OK' if reading[2] == 'ok' else 'NA'
+        health_desc = 'good' if status == 'OK' else 'bad'
+        recommendation = sspl_constants.DEFAULT_ALERT_RECOMMENDATION if status != 'OK' else 'NA'
+        resp = {
+            "uid": uid,
+            "fru": "false",
+            "last_updated":  int(time.time()),
+            "health": {
+                "status": status,
+                "description": f"{uid} is in {health_desc} health",
+                "recommendation": f"{recommendation}",
+                "specifics": [
+                    {
+                        "Sensor Reading": f"{reading[-1]}",
+                        "lower_critical_threshold": lower_critical,
+                        "upper_critical_threshold": upper_critical,
```

Comment:
>Upper Non-Recoverable : 13.430
Lower Non-Recoverable : 10.180

lets put these two too in specifics, are imp from threshold perspective